libraries/AP_RangeFinder/AP_RangeFinder_LeddarOne.h

```diff
@@ -74,4 +74,6 @@ class AP_RangeFinder_LeddarOne : public AP_RangeFinder_Backend
     LeddarOne_ModbusStatus modbus_status = LEDDARONE_MODBUS_STATE_INIT;
     uint8_t read_buffer[LEDDARONE_READ_BUFFER_SIZE];
     uint32_t read_len;
+
+    uint8_t send_request_buffer[8] = {0};
```

Comment:
I like this moving here to save the work but would be even more helpful if it was const. Is that possible with the CRC calc in the constructor? Maybe if the crc was static?